src/hooks/useMixpanel.ts

```diff
@@ -201,14 +200,20 @@ export default function useMixpanel(trade?: Aggregator | undefined, currencies?:
           mixpanel.track('Wallet Connected')
           break
         case MIXPANEL_TYPE.SWAP_INITIATED: {
+          const { gasUsd, inputAmount, priceImpact } = payload as {
```

Comment:
Can `payload` undefined?
```suggestion
          const { gasUsd, inputAmount, priceImpact } = (payload || {}) as {
```

Reply:
Agree. I fixed it